h5p-bildetema/src/components/LanguageSelector/LanguageSelector.module.scss

```diff
@@ -0,0 +1,9 @@
+.languageSelector {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.language_select {
+  width: 50%;
```

Comment:
This `width: 50%` might become a nuisance in the future, because the elements don't scale gracefully. I think using `column-count: 2` on the parent element is more defensive CSS, and makes keyboard users traverse the options column by column, which especially helps visually impaired users.